app/src/main/res/drawable/ic_search_foreground.xml

```diff
@@ -0,0 +1,15 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You can just remove `_foreground` from the name and use it like that :D. 

Also the `xyz_background.xml` can be deleted together with all the mip-map files